prowler/providers/aws/lib/security_hub/security_hub.py

```diff
@@ -16,6 +18,19 @@
 SECURITY_HUB_MAX_BATCH = 100
 
 
+@dataclass
```

Comment:
Do you need this? The `Connection` class inherits from `BaseModel`.

Reply:
Yep, it is needed :)